src/parser/Statement.ts

```diff
@@ -719,19 +719,17 @@ export class StopStatement extends Statement {
 
 export class ForStatement extends Statement {
     constructor(
-        readonly tokens: {
```

Comment:
Removed the `tokens` collection in favor of direct token properties (to be more consistent with rest of AST in bsc).

Reply:
Most of the churn in this function is renamed properties